src/Build/BackEnd/Components/SdkResolution/SdkResolverService.cs

```diff
@@ -232,14 +391,33 @@ private void Initialize(LoggingContext loggingContext, ElementLocation location)
         {
             lock (_lockObject)
             {
-                if (_resolvers != null)
+                if (_resolversList != null)
                 {
                     return;
                 }
 
                 MSBuildEventSource.Log.SdkResolverServiceInitializeStart();
-                _resolvers = _sdkResolverLoader.LoadResolvers(loggingContext, location);
-                MSBuildEventSource.Log.SdkResolverServiceInitializeStop(_resolvers.Count);
+                _resolversList = _sdkResolverLoader.LoadResolvers(loggingContext, location);
+                MSBuildEventSource.Log.SdkResolverServiceInitializeStop(_resolversList.Count);
```

Comment:
Does this event mean the same thing any more? Is it now the span over loading manifests but not assemblies, where before it included assemblies?

Reply:
Ah, it does mean the same thing but it isn't called all the time now. Can you please add events for the other path?